packages/babel-helper-fixtures/src/index.ts

```diff
@@ -509,15 +514,8 @@ export function multiple(entryLoc: string, ignore?: Array<string>) {
 }
 
 export function readFile(filename: string | undefined) {
-  try {
-    if (filename === undefined) {
-      return "";
-    }
+  if (filename !== undefined && fs.existsSync(filename)) {
     return fs.readFileSync(filename, "utf8").trimRight();
-  } catch (e) {
-    if (e.code === "ENOENT") {
-      return "";
-    }
-    throw e;
```

Comment:
The previous try-catch pattern is recommended by the Node.js official docs: https://nodejs.org/docs/latest-v24.x/api/fs.html#fsexistspath-callback

> In general, check for the existence of a file only if the file won't be used directly, for example when its existence is a signal from another process.